Hovering community in the profile showcase leads to crash

# Bug Report

## Description

https://github.com/status-im/status-desktop/assets/2522130/f399b4e3-4080-463a-8381-f41e84f6d77c


## Steps to reproduce

1. Open profile for a contact
2. Hover some community in the contact's profile showcase

#### Expected behavior

No segfault

#### Actual behavior

Segfault

### Additional Information

Current master: 1884f6e03378f7c1ca713e6591c304bb8083b47f

> @MishkaRogachev try again pls with latest master, I _think_ I fixed this with my PR #14564

Hmm, just updated and tried again. Crashes for me with: 
```
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [run-macos] Segmentation fault: 11
```

--- a/ui/imports/shared/views/profile/ProfileShowcaseCommunitiesView.qml
+++ b/ui/imports/shared/views/profile/ProfileShowcaseCommunitiesView.qml
@@ -61,9 +61,6 @@ Item {
             asset.height: 32
             name: model.name ?? ""
             memberCountVisible: false
-            layer.enabled: hovered
-            border.width: hovered ? 0 : 1
-            border.color: Theme.palette.baseColor2
             banner: model.bannerImageData ?? ""
             descriptionFontSize: 12
             descriptionFontColor: Theme.palette.baseColor1

